monitoring/.env.example

```
@@ -0,0 +1,44 @@
+PROMETHEUS_PORT=9090
+# TSDB retention window.
+PROMETHEUS_RETENTION=15d
+# Default scrape interval (also used as evaluation_interval).
+PROMETHEUS_SCRAPE_INTERVAL=15s
+
+# ─── Backend scrape target ──────────────────────────────────────────────────
+# host.docker.internal resolves to the Docker host (macOS native;
+# Linux via the host-gateway alias declared in docker-compose.yml).
+# In Coolify, set this to the backend service DNS name, e.g. steam5-backend:8081.
+STEAM5_METRICS_TARGET=host.docker.internal:8081
+STEAM5_METRICS_SCHEME=http
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The STEAM5_METRICS_SCHEME key should go before the STEAM5_METRICS_TARGET key

(UnorderedKey)